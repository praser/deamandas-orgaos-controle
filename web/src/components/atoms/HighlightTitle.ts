import styled from 'styled-components';

const HighlightTitle = styled.h3`
  /* Text */
  font-family: Nunito, -apple-system, system-ui, 'Segoe UI', Roboto,
    'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
    'Segoe UI Symbol', 'Noto Color Emoji';
  font-size: 11.2px;
  font-weight: 700;
  line-height: 16.8px;
  text-align: left;
  text-size-adjust: 100%;
  text-transform: uppercase;

  /* Appearance */
  color: rgb(70, 115, 223);

  /* Other */
  overflow-wrap: break-word;
`;

export default HighlightTitle;
