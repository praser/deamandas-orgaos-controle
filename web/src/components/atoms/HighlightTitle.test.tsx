import React from 'react';
import { render, screen } from '@testing-library/react';
import HighlightTitle from './HighlightTitle';

describe('HighlightTitle component', () => {
  let highlighttitle: HTMLElement;
  const testId = 'testid';

  beforeEach(() => {
    render(<HighlightTitle data-testid={testId} />);
    highlighttitle = screen.getByTestId(testId);
  });

  it('is expected to match the snapshot', () => {
    expect(highlighttitle).toMatchSnapshot();
  });
});
