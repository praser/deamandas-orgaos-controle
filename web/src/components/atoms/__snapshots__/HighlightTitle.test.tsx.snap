// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HighlightTitle component is expected to match the snapshot 1`] = `
<h3
  class="sc-bdvvaa eouGHb"
  data-testid="testid"
/>
`;
